charts/velero/ci/test-values.yaml

```diff
@@ -1,7 +1,7 @@
 # Set provider name and backup storage location bucket name
 configuration:
   backupStorageLocation:
-  - name: backups-primary
+  - name: default
```

Comment:
I will change it back to `backups-primary` in a later version, for Velero v1.13.0, I modified the default BSL selections logic:
- Creating one BSL with the name of `default`
- Not Creating the name of `default` BSL but specifying one default BSL existing name such as `backups-primary` in the Velero server startup command.

Or it would get an error when the Velero server starts. Here is the [issue](https://github.com/vmware-tanzu/velero/issues/7359)